api/tests/identity-access-management/integration/domain/services/refresh-token-service.test.js

```diff
@@ -34,6 +35,90 @@ describe('Integration | Identity Access Management | Domain | Service | refresh-
         const refreshTokensInDb = await refreshTokenService.findByUserId(userId);
         expect(refreshTokensInDb).to.deep.equal(['123:pix-orga:XXX-123-456']);
       });
+      context('set refresh token expiration by scope', function () {
+        let userId, source, uuidGenerator;
+        let refreshTokenTemporaryStorage, userRefreshTokensTemporaryStorage;
+
+        beforeEach(async function () {
+          userId = '123';
+          source = 'APP';
+          uuidGenerator = () => 'XXX-123-456';
+
+          refreshTokenTemporaryStorage = {
+            save: sinon.stub(),
+          };
+
+          userRefreshTokensTemporaryStorage = {
+            lpush: sinon.stub(),
+            expire: sinon.stub(),
+          };
+        });
+
+        it('sets a default refresh token lifespan when scope is not defined', async function () {
+          // given
+          const refreshTokenLifespanMs = 3600000;
+          sinon.stub(config.authentication, 'refreshTokenLifespanMs').value(refreshTokenLifespanMs);
+
+          // when
+          await refreshTokenService.createRefreshTokenFromUserId({
+            userId,
+            source,
+            uuidGenerator,
+            refreshTokenTemporaryStorage,
+            userRefreshTokensTemporaryStorage,
+          });
+
+          // then
+          expect(refreshTokenTemporaryStorage.save.args[0][0].expirationDelaySeconds).to.be.equal(
+            refreshTokenLifespanMs / 1000,
+          );
+        });
+        it('sets the app refresh token lifespan when scope is defined', async function () {
+          // given
+          const scope = 'mon-pix';
+
+          const refreshTokenLifespanMs = 1800000;
+          sinon.stub(config.authentication.refreshTokenLifespanMsByScope, 'mon-pix').value(refreshTokenLifespanMs);
+
+          // when
+          await refreshTokenService.createRefreshTokenFromUserId({
+            userId,
+            source,
+            scope,
+            uuidGenerator,
+            refreshTokenTemporaryStorage,
+            userRefreshTokensTemporaryStorage,
+          });
+
+          // then
+          expect(refreshTokenTemporaryStorage.save.args[0][0].expirationDelaySeconds).to.be.equal(
+            refreshTokenLifespanMs / 1000,
+          );
+        });
+        it('sets the default token lifespan when scope value is not found', async function () {
```

Comment:
Petite suggestion de formulation : 
```suggestion
        it('sets the default token lifespan when scope value is not defined in configuration', async function () {
```